contracts/payment/SplitPayment.sol

```diff
@@ -12,7 +12,7 @@ contract SplitPayment {
   using SafeMath for uint256;
 
   uint256 public totalShares = 0;
-  uint256 public totalReleased = 0;
+  uint256 public totalFundsPerShare = 0;
```

Comment:
I'd make this a `pivate` variable, as well as `totalReleased` if we add it back in.